cns/kubecontroller/nodenetworkconfig/conversion_windows_test.go

```diff
@@ -13,21 +13,69 @@ var validOverlayRequest = &cns.CreateNetworkContainerRequest{
 			PrefixLength: uint8(subnetPrefixLen),
 			IPAddress:    primaryIP,
 		},
-		GatewayIPAddress: "10.0.0.0",
+		GatewayIPAddress: "10.0.0.1",
 	},
 	NetworkContainerid:   ncID,
 	NetworkContainerType: cns.Docker,
 	SecondaryIPConfigs: map[string]cns.SecondaryIPConfig{
-		"10.0.0.1": {
-			IPAddress: "10.0.0.1",
-			NCVersion: version,
-		},
 		"10.0.0.2": {
 			IPAddress: "10.0.0.2",
+			NCVersion: 0,
+		},
+	},
+}
+
+var validVNETBlockRequest = &cns.CreateNetworkContainerRequest{
```

Comment:
can you please also add a test case for ipv6?

Reply:
we'll add later since this needs to go in for quick-fix